content/geometry/HullDiameter.h

```diff
@@ -0,0 +1,22 @@
+/**
+ * Author: Oleksandr Bacherikov, chilli
+ * Date: 2019-05-05
+ * License: CC0
+ * Source: https://codeforces.com/blog/entry/48868
+ * Description: Returns the two points with max distance on a convex hull. If input isn't a convex hull, call "convexHull" first.
```

Comment:
are colinear points allowed? duplicate points?

Reply:
They aren't allowed - added a note in the description.